_sass/_base.scss

```diff
@@ -33,15 +34,15 @@ body {
   line-height: 1.6;
 
   @media (min-width: $mq-xs) {
-    font-size: 102.5%;
+    font-size: 133.25%;
```

Comment:
Why are font sizes changed this drastically? I'm not going to merge this in.

Reply:
In my opinion, headings look nicer on my website (see template post). I'm not much experienced with scss/css, would changing the font-size in % in the following style change headings size significantly?

```css
  @media (min-width: $mq-xs) {
    font-size: 133.25%;
  }

  @media (min-width: $mq-sm) {
    font-size: 136%;
  }

  @media (min-width: $mq-md) {
    font-size: 140%;
  }

  @media (min-width: $mq-lg) {
    font-size: 110%;
  }

  @media (min-width: $mq-xl) {
    font-size: 115%;
  }

  @media (min-width: $mq-xxl) {
    font-size: 125%;
  }
}
```
If the answer is yes, does that mean that you think the headings are two big on the template post? Anyway, I can remove them if you'd like to merge the PR, let me know!

Reply:
I think this is the last change request, can I resolve the conversation?